packages/augur-sdk/src/connector/ww-connector.ts

```diff
@@ -0,0 +1,57 @@
+import RunWorker from "../state//Sync.worker";
+
+import { API } from "../state/api/API";
+import { Augur } from "../Augur";
+import { BlockAndLogStreamerListener } from "../state//db/BlockAndLogStreamerListener";
+import { Connector, Callback } from "./connector";
+import { ContractDependenciesEthers } from "contract-dependencies-ethers";
+import { DB } from "../state//db/DB";
+import { EthersProvider } from "@augurproject/ethersjs-provider";
+import { EventLogDBRouter } from "../state//db/EventLogDBRouter";
+import { JsonRpcProvider } from "ethers/providers";
+import { PouchDBFactory } from "../state//db/AbstractDB";
+import { UploadBlockNumbers, Addresses } from "@augurproject/artifacts";
+import { Controller } from "../state/Controller";
+
+const settings = require("@augurproject/sdk/src/state/settings.json");
+
+export class WebWorkerConnector extends Connector {
+  private api: API;
+  private worker: any;
+
+  public async connect(params?: any): Promise<any> {
+    const ethersProvider = new EthersProvider(new JsonRpcProvider(settings.ethNodeURLs[4]), 10, 0, 40);
```

Comment:
All of this stuff is repeated in Sync.ts and here. Could be refactored. Here it exists just to get a db.